Server/Communication/router.py

```diff
@@ -114,6 +114,7 @@ def processRequest(message):
 			  "LGN":User.routehelper.login,	
 			  "PA":Communication.routehelper.pingAuthentication,
 			  "ST":Game.routeunithelper.setTeam,
+			  "GUI":User.routehelper.getUserInfo,
```

Comment:
Let's aim for alphabetical command inputs, for easier searching later when we (potentially) have a large list of commands.  Obviously minor nit, but more importantly this command needs an entry in the comment table above.